utils/types/priority.go

```diff
@@ -108,3 +108,19 @@ func (p *Priority) Scan(value interface{}) error {
 func (p *Priority) Value() (driver.Value, error) {
 	return string(*p), nil
 }
+
+func ScoreToPriority(score float32) Priority {
```

Comment:
I was following https://www.first.org/cvss/specification-document#i5.
It is similar to https://nvd.nist.gov/cvss.cfm.